src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(DifferentColors.values().length);
+        DifferentColors color = DifferentColors.values()[index];
+        return color.toString();
```

Comment:
Consider using `color.name()` instead of `color.toString()` for a more reliable string representation of the enum constant. This ensures that the string representation is consistent with the enum's name.